src/data/objects/managedlinodesetting.yaml

```diff
@@ -0,0 +1,51 @@
+# vim: set et ts=2 sw=2 tw=80 :
+name: ManagedLinodeSetting
+prefix: managed/linode_settings
+description: >
+  Managed Linode Setting objects describe SSH information for your managed Linodes.
+schema:
+  id:
+    type: Integer
+    value: 123
+    description: The ID of the Linode's settings.
+  label:
+    type: String
+    editable: false
+    value: LinodeSettingsLabel
+    description: >
+      Unique label of your Linode.
+  group:
+    type: String
+    editable: false
+    value: ManagedLinodeGroup
+    description: >
+      This Linode's display group.
+  ssh:
+    type: Object
+    description: >
+      The SSH settings for your Linode.
+    user:
+      type: String
+      editable: true
+      value: root
+      description: >
+        The SSH username used to log into your Linode.
+    ip:
+      type: String
+      editable: true
+      value: 204.23.61.122
+      description: >
+        One of "any", IPv4, or IPv6 address to use to SSH into the Linode.
```

Comment:
This phrasing feels weird. Maybe
`Either "any" or a valid IP address. Both IPv4 and IPv6 are supported`
or something like that. Lets ask the team

Reply:
They can't be any old valid IP address. They have to be associated with the Linode specifically.

Reply:
Lets call that out too then